syft/workers/base.py

```diff
@@ -117,6 +122,15 @@ def _recv_msg(self, message):
 
         """
         raise NotImplementedError  # pragma: no cover
+        
+    def Load_Data(self,Data):
```

Comment:
I think all data will be garbage collected automatically (deleted) after this method runs. Instead each tensor should be registered (i.e., don't use .send(), use register_obj())